### PR TITLE
Improve gitlab detection

### PR DESCRIPTION
**Pull request by jvoisin**
This can be tested on [this repo](https://0xacab.org/mat/).

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -2645,6 +2645,10 @@
 			"headers": {
 				"Set-cookie": "_gitlab_session"
 			},
+			"html": [
+				"<meta content=\"GitLab\" property=\"og:site_name\">",
+				"<meta content=\"https?://[^/]+/assets/gitlab_logo-"
+			],
```
Comment (AliasIO) [inline src/apps.json]: Minor point, you can use `meta` instead of `html` for these.

Comment (jvoisin) [inline src/apps.json]: Because those `meta` tags have no names, only ` content` and ` property`.

**Comment by jvoisin** [comment]
Do we want to add `property` as a key to the `meta` dict? Currently, only `name` is the only considered key.

**Comment by twindual** [comment]
Vote for "yes" from me. It will allow checking for "mobile friendly"
options, OpenGraph protocol, and a few others that use property.
It think its better than having an HTML check or double checking every name
against property.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**